cfme/common/provider_views.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+from widgetastic.widget import View, Text
+
+from cfme import BaseLoggedInPage
+from widgetastic_patternfly import Dropdown
+from widgetastic_manageiq import DetailsToolBarViewSelector
+from widgetastic_manageiq import (BreadCrumb,
+                                  SummaryTable,
+                                  Button,
+                                  TimelinesView)
+
+
+class ProviderDetailsToolBar(View):
+    """
+    represents provider toolbar and its controls
+    """
+    monitoring = Dropdown(text='Monitoring')
+    configuration = Dropdown(text='Configuration')
+    reload = Button(title='Reload Current Display')
+    policy = Dropdown(text='Policy')
+    authentication = Dropdown(text='Authentication')
+
+    @View.nested
+    class view_selector(DetailsToolBarViewSelector):  # NOQA
+        pass
```

Comment:
You could possibly replace this with simpler `view_selector = View.nested(DetailsToolBarViewSelector)`, I believe this construciton should work too.